mapusaurus/respondents/views.py

```diff
@@ -51,8 +51,10 @@ def select_metro(request, agency_id, respondent):
     """Once an institution is selected, search for a metro"""
     institution = get_object_or_404(Institution, respondent_id=respondent,
                                     agency_id=int(agency_id))
+    year = request.session.get('year', 2014)
```

Comment:
:+1: we can probably pull the largest year from a table somewhere, so we don't have to hardcode it anywhere.